torch/csrc/jit/codegen/cuda/fusion.cpp

```diff
@@ -214,6 +214,20 @@ void Fusion::print() {
   std::cout << "}\n";
 }
 
+void Fusion::printValuesMap() {
```

Comment:
Would it make sense in our printers to add an output/warning if nothing will be printed like "No value map entries found."

Reply:
I was thinking a small message like "Warning: No values in map" between the lines would be a bit more intuitive.